spec/ordinary-and-exotic-object-behaviours.html

```diff
@@ -0,0 +1,326 @@
+          1. If Type(_P_) is a Symbol, return *false*.
+          1. Return ! RecordHasProperty(_R_, _P_).
+        </emu-alg>
+      </emu-clause>
+            
+      <emu-clause id="sec-record-exotic-objects-get-p-receiver">
+        <h1>[[Get]] ( _P_, _Receiver_ )</h1>
+        <p>When the [[Get]] internal method of a Record exotic object _R_ is called with property key _P_ and ECMAScript language value _Receiver_, the following steps are taken:</p>
+        <emu-alg>
+          1. Assert: IsPropertyKey(_P_) is *true*.
+          1. If Type(_P_) is a Symbol, return *undefined*.
+          1. Return ! RecordGet( _R_, _P_ ).
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-record-exotic-objects-set-p-receiver">
+        <h1>[[Set]] ( _P_, _Receiver_ )</h1>
+        <p>When the [[Set]] internal method of a Record exotic object _R_ is called with property key _P_ and ECMAScript language value _Receiver_, the following steps are taken:</p>
+        <emu-alg>
+          1. Return *false*.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-record-exotic-objects-delete-p">
+        <h1>[[Delete]] ( _P_ )</h1>
+        <p>When the [[Delete]] internal method of a Record exotic object _R_ is called with property key _P_, the following steps are taken:</p>
+        <emu-alg>
+          1. Assert: IsPropertyKey(_P_) is *true*.
+          1. If Type(_P_) is a Symbol, return *true*.
+          1. Return *false*.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-record-exotic-objects-ownpropertykeys">
+        <h1>[[OwnPropertyKeys]] ( )</h1>
+        <p>When the [[OwnPropertyKeys]] internal method of a Record exotic object _R_ is called, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _keys_ be a new empty List.
+          1. Let _rec_ be _R_.[[RecordData]].
+          1. Assert: Type(_rec_) is Record.
+          1. Let _len_ be the length of _rec_.
+          1. For each pair _p_ of _rec_.[[Fields]], do
+            1. Let _key_ be _p_.[[Key]]
+            1. Assert: Type(_key_) is String
+            1. Add _key_ to _keys_
+          1. Return _keys_
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-recordcreate" aoid="RecordCreate">
+        <h1>RecordCreate ( _value_ )</h1>
+        <p>This abstract operation RecordCreate takes a _value_ argument. It is used to specify the creation of new Record exotic objects. It performs the following steps when called:</p>
+        <emu-alg>
+            1. Assert: Type(_value_) is a Record.
+            1. Assert: Type(_value_.[[Fields]]) is a List.
+            1. Let _R_ be ! MakeBasicObject(&laquo; [[RecordData]] &raquo;).
+            1. Set _R_'s essential internal methods to the definitions specified in <emu-xref href="#sec-record-exotic-objects"></emu-xref>.
+            1. Set _R_.[[Prototype]] to *null*.
+            1. Set _R_.[[RecordData]] to _value_.
+            1. OrdinaryPreventExtensions(_R_).
```

Comment:
This line should be deleted; it doesn't make sense (since Record wrappers are not ordinary objects) and is unnecessary (since it already implements the extension-related methods appropriately).